main.tf

```diff
@@ -0,0 +1,86 @@
+locals {
+  ecr_need_policy = length(var.principals_readonly_access) > 0 ? true : false
+  policy_rule_remove_expired_image = [{
+    rulePriority = 2
+    description  = "Expire images older than ${var.max_image_days} days",
+    selection = {
+      tagStatus = "any"
```

Comment:
In this case, isn't the policy going to expire any image older than `var.max_image_days` including the `latest` image? I'm just thinking in the case of no new image been built before we reach `var.max_image_days` and all the images get deleted...or am I mistaken here?

Reply:
Hey yes you are right, I can include a statement to ignore certain tags, but I think its better to remove this now and add this in when needed :-)